packages/frontend/src/components/common/Table/TableProvider.tsx

```diff
@@ -100,11 +100,12 @@ export const TableProvider: FC<React.PropsWithChildren<ProviderProps>> = ({
             ...rowColumn,
             meta: {
                 ...rowColumn.meta,
-                className: 'sticky-column first-sticky-column',
+                className: 'sticky-column',
                 width: rowColumnWidth,
                 style: {
                     maxWidth: rowColumnWidth,
                     minWidth: rowColumnWidth,
+                    backgroundColor: 'white',
```

Comment:
this is the column 
<img width="75" alt="image" src="https://github.com/user-attachments/assets/6e130aa3-a119-4d07-bc31-3a6e2dde6507" />